lib/Runtime/Debug/DebugContext.cpp

```diff
@@ -142,11 +142,7 @@ namespace Js
         });
         END_TRANSLATE_OOM_TO_HRESULT(hr);
 
-        if (hr != S_OK)
-        {
-            Assert(FALSE);
-            return hr;
-        }
+        DEBUGGER_ATTACHDETACH_FATAL_ERROR_IF_FAILED(hr);
```

Comment:
why failfast when it is OOM? Why not replace with Assert(hr==S_OK || hr==E_OUTOFMEMORY)?